xunit.runner.visualstudio.nuspec

```diff
@@ -41,5 +41,8 @@ Supported platforms:
     <file src="build\xunit.runner.visualstudio.wpa81.targets"                                   target="build\wpa81\xunit.runner.visualstudio.targets" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.dll" target="build\wpa81\" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.pri" target="build\wpa81\" />
+
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.deps.json" target="build\netstandard1.5\" />
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.dll" target="build\netstandard1.5\" />
```

Comment:
Targetting to netstandard1.5 will make it to support .NET Framework 4.6.2 or later and .NET Core 1.0 both, so I kept it as netstandard1.5 instead of netcoreapp1.0.
What's your recommendation? Should we change it to netcoreapp1.0 or classify this change as netstandard support?